src/components/toast/toast.stories.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { withKnobs, text, select } from "@storybook/addon-knobs";
+
+
+import readme from './readme.md';
+
+export default {
+  title: 'Toast',
+  decorators: [withKnobs],
+  parameters: {
+    notes: { markdown: readme },
+  }
+};
+
+export const toastSystem = () => {
+  const variantOptions = { System: 'system', Error: 'error', Success: 'success', Warning: 'warning' };
+  const actionTypeOptions = { Button: 'button', Icon: 'icon' };
+  const buttonActionOptions = { Close: 'close', Custom: 'custom' };
+
+  const showToast = async () => {
+    const newtoast = document.createElement('bds-toast');
+    const toastContainer = document.querySelector("toast-container").shadowRoot;
+    toastContainer.appendChild(newtoast);
+
+    const icon = text('icon', 'info');
+    const variant = select('variant', variantOptions, 'system');
+    const actionType = select('action-type', actionTypeOptions, 'button');
+    const toastTitle = text('toast-title', 'Toast Title');
+    const toastText = text('toast-text', 'Lorem ipsum');
+    const buttonText = text('button-text','Cancelar');
+    const duration = text('duration', 0);
```

Comment:
Would be nice if you put all those default values into const variables

Reply:
we can discuss it, but because it is a story, i think it's not necessary.